### PR TITLE
add the scripts to run the build, test locally

### PR DESCRIPTION
**Pull request by xuzhg**
### Issues

* add the scripts to run the build, test locally

### Description

Now, it can run classic & .net core test.

image

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+	Write-Host
+    $script:BUILD_END_TIME = Get-Date
+}
+
+Function RunTest($title, $testdir, $framework)
+{
+    Info("Running test $title...")
+    if ($framework -eq 'dotnet')
+    {
+		$Conf = "/p:Configuration=" + "$Configuration"
```
Comment (AlanWong-MS) [inline build.ps1]: nit: extra indent

File: build.cmd [hunk 2]
```diff
@@ -1,72 +1,2 @@
...
-REM Continue to support original switches for those
-REM who might have scripts setup to use them.
-if /I "%1" == "E2EV4" goto BuildE2EV4
-if /I "%1" == "FULL" (
-  set FullBuild=1
-  goto BuildDefaults
-)
-
-REM Build a specified target
-%MSBuild% WebApiOData.msbuild /m /nr:false /t:%* /p:Platform="Any CPU" /p:Desktop=true /v:M /fl /flp:LogFile=bin\msbuild.log;Verbosity=Normal
```
Comment (robward-ms) [inline build.cmd]: Still need these?

File: build.cmd [hunk 3]
```diff
@@ -1,72 +1,2 @@
...
-
-REM Build product code and unit tests
-:BuildDefaults
-%MSBuild% WebApiOData.msbuild /m /nr:false /p:Platform="Any CPU" /p:Desktop=true /v:M /fl /flp:LogFile=bin\msbuild.log;Verbosity=Normal
-if %ERRORLEVEL% neq 0 goto BuildFail
-if %FullBuild% neq 0 goto BuildE2EV4
-goto BuildSuccess
-
-REM Build product and V4 End to End tests
-:BuildE2EV4
```
Comment (robward-ms) [inline build.cmd]: Still need these?

File: build.cmd [hunk 4]
```diff
@@ -1,72 +1,2 @@
...
-)
-
-REM Build a specified target
-%MSBuild% WebApiOData.msbuild /m /nr:false /t:%* /p:Platform="Any CPU" /p:Desktop=true /v:M /fl /flp:LogFile=bin\msbuild.log;Verbosity=Normal
-if %ERRORLEVEL% neq 0 goto BuildFail
-goto BuildSuccess
-
-REM Build product code and unit tests
-:BuildDefaults
-%MSBuild% WebApiOData.msbuild /m /nr:false /p:Platform="Any CPU" /p:Desktop=true /v:M /fl /flp:LogFile=bin\msbuild.log;Verbosity=Normal
```
Comment (robward-ms) [inline build.cmd]: Still need these?

Comment (xuzhg) [reply build.cmd]: You mean "WebApiOData.msbuild" file itself?. If the script are ok. I will remove the "WebApiOData.msbuild" file.

File: build.ps1 (hunk 1 above)
Comment (xuzhg) [reply build.ps1]: Thanks. Will change it using space.

File: build.cmd (hunk 4 above)
Comment (robward-ms) [reply build.cmd]: Yes, I meant the MSBuild files. They should be deleted if they are no longer needed.

**Comment by xuzhg** [comment]
Close it because the similar changes merged in PR: #1433 Thanks!

**Pull request closed without merging**